client/state/ui/guided-tours/test/config.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import { isNewUser } from 'state/ui/guided-tours/selectors';
+import { getSectionName } from 'state/ui/selectors';
+
+const tours = {
+	main: {
+		meta: {
+			version: 'test',
+			path: '/',
+			context: state => isNewUser( state ),
```

Comment:
Worth noting that the tests were failing when this line read:

```js
context: isNewUser
```

I believe this is due to circular dependencies between `test/config` and `selectors`. Wrapping `isNewUser` in a lambda solves this by giving us lazy evaluation of `isNewUser`.